test/test_issues/test_issue166.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+import rdflib
+from rdflib import Graph
+
+
+def test_issue_166() -> None:
```

Comment:
Just briefly, I would much rather prefer this goes into https://github.com/RDFLib/rdflib/blob/eba13739e1d24b3e52697a1ec9545a361116951f/test/test_sparql/test_sparql.py#L694-L727